Update docs/strip page content (#2188)
vanilla-framework/vanilla-framework: docs/en/patterns/strip.md

<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/strips/strips-light/"
    class="js-example">
View example of the strip light pattern
</a>

<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/strips/strips-dark/"
    class="js-example">
View example of the strip dark pattern
</a>

### Accent strip

> The purpose of the strip accent pattern displays content with a highlighted site accent strip style.

Reviewer: ```suggestion
The purpose of the strip accent pattern is to display content with a highlighted strip using the accent colour.
```

<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/strips/accent/"
  class="js-example">
View example of the pattern strip accent
</a>

### Image strip

This pattern allows for an image background to be appear as a background on a strip.

<div class="p-notification--information">
  <p class="p-notification__response">

>     <span class="p-notification__status">Note:</span>Declare the background-image as an inline style attribute in the HTML markup.

Reviewer: HTML Markup == "Hypertext Markup Language Markup" :P
```suggestion
    <span class="p-notification__status">Note:</span>Declare the background-image as an inline style attribute in the HTML.
```
